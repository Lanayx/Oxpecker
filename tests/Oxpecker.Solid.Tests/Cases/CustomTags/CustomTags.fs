module Oxpecker.Solid.Tests.Cases.CustomTags

open System
open Oxpecker.Solid
open Oxpecker.Solid.Tests.Cases.MyTag

[<SolidComponent>]
let Test () =
    MyTag(myAttr = "foo") {
        "Hello " + DateTime().Day.ToString()
        MyTag()
    }
