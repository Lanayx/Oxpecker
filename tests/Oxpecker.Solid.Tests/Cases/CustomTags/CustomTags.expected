import { ImportedName } from "ImportedModule";
import { minValue, day } from "./fable_modules/fable-library-js.4.24.0/Date.js";
import { int32ToString } from "./fable_modules/fable-library-js.4.24.0/Util.js";

export function Test2() {
    return <ImportedName x="y"
        a="b">
        <ImportedName x="y"
            a="b"></ImportedName>
        Hello
    </ImportedName>;
}

export function Test1() {
    let copyOfStruct_1;
    return <my-tag myAttr="foo">
        {"Hello " + ((copyOfStruct_1 = day(minValue()), int32ToString(copyOfStruct_1)))}
        <my-tag></my-tag>
    </my-tag>;
}

